pkg/nozzle/okta/okta.go

```diff
@@ -62,6 +63,16 @@ func (Driver) New(opts map[string]string) (nozzle.Nozzle, error) {
 		return nil, fmt.Errorf("okta nozzle requires 'subdomain' config parameter")
 	}
 
+	// check if operator entered full domain instead of subdomain
+	if strings.Contains(subdomain, ".okta.com") {
+		domains := strings.Split(subdomain, ".")
+		// i don't think sub-subdomains are legal for okta, but i could be wrong
```

Comment:
also before we merge to master can you delete this and if it represents more code/testing we need to do, create a ticket that references this line?